src/roiextractors/imagingextractor.py

```diff
@@ -147,11 +112,28 @@ def get_frame_shape(self) -> Tuple[int, int]:
 
         Notes
         -----
-        This method is equivalent to get_image_shape() and is provided for consistency
-        with the naming convention used in volumetric extractors.
+        This method is equivalent to get_image_shape()
         """
         return self.get_image_shape()
 
+    def get_sample_shape(self) -> Union[Tuple[int, int], Tuple[int, int, int]]:
+        """
+        Get the shape of a single sample elements from the series.
+
+        If the series is volumetric, the shape is the shape of the volume and otherwise
+        returns the shape of a single frame/image.
+
+        Returns
+        -------
+        sample_shape: tuple
+            Shape of a single sample from the time series (num_rows, num_columns, num_planes)
+            if volumetric (num_rows, num_columns) otherwise
+        """
+        if self.is_volumetric:
+            return self.get_volume_shape()
+        else:
+            return self.get_frame_shape()
+
```

Comment:
Thanks @h-mayorquin, I think this looks good. In the PR description you mention that `get_sample_shape` returns `get_image_shape` for planar extractor, but it actually returns `get_frame_shape`. 

I probably missed when we added  `get_image_shape` and  `get_frame_shape`. So the idea there is that each imaging extractor would implement their own `get_image_shape` but you could still call the base `get_frame_shape`?

Reply:
I think that this question and @alessandratrapani one below are related.

So, first here https://github.com/catalystneuro/roiextractors/issues/394 we changed `get_image_size` to `get_image_shape`. But then in #406 we changed to use the following terminology: frame for planar, volume for volumetric and sample in general. 

I am not sure how to organize this better. Maybe we should get rid of `get_image_shape` (that we just introduced, I know) to be super consistent? image_shape and frame_shape both seem super clear on their and intent but it would be less code to maintain. Anyway, I made a lot of changes recently and did not have time to push for this. We can move this forward in another PR or leave it as it is.

Reply:
Yeah it definitely confused me when reviewing, but we can put this in the backlog and come back to it later.

Reply:
Do you think that we should remove `get_image_shape` and leave only `get_frame_shape` to avoid confusions?